app/safecast/tasks/device_stories/update_metadata.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
```

Comment:
I renamed this task from `Tasks::DeviceStory::UpdateMetadata` to `Tasks::DeviceStories::UpdateMetadata` to avoid class naming conflicts with `DeviceStory` Active record